builder/amazon/common/cli_config.go

```diff
@@ -1,159 +0,0 @@
-package common
```

Comment:
Yes, it's dead code now. Most of the code in here was to try to handle things now handled by the sdk.